enhancements/ingress/haproxy-thread-tuning.md

```diff
@@ -0,0 +1,190 @@
+---
+title: haproxy-thread-tuning
+authors:
+  - "@rfredette"
+reviewers:
+  - "@Miciah"
+  - "@danehans"
+  - "@frobware"
+  - "@sgreene"
+  - "@knobunc"
+  - "@miheer"
+  - "@candita"
+approvers:
+  - "@knobunc"
+  - "@miciah"
+  - "@frobware"
+  - "@danehans"
+creation-date: 2021-03-24
+last-updated: 2021-03-25
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
+see-also:
```

Comment:
May want to include https://github.com/openshift/enhancements/pull/449 (Tunable router buffers) in "see-also".